compiler/semcall.nim

```diff
@@ -212,7 +210,6 @@ proc resolveOverloads(c: PContext, n, orig: PNode,
       if result.state != csMatch:
         n.sons.delete(1)
         orig.sons.delete(1)
-        excl n.flags, nfExprCall
```

Comment:
I got the impression that the "changed files" summary for the whole pull request also included that line, but now I see that's not the case. 

I now remember what happened. One of the merges was having conflicts all over the place in semcall up, so I stared from my own code and carefully migrated all the changes from the devel branch by hand (line by line):
https://github.com/nim-lang/Nim/pull/5606/commits/d6d548c32f4d60bd4f89c377fb94bfc57ad4cbfd